.evergreen/config.yml

```diff
@@ -680,10 +680,11 @@ axes:
   - id: libmongoc-version
     display_name: libmongoc version
     values:
-      - id: "1.16-latest"
-        display_name: "1.16 latest"
-        variables:
-          LIBMONGOC_VERSION: "r1.16"
+# Todo: re-enable once 1.17 is branches
```

Comment:
"is branched"

Also, perhaps you want to pre-emptively update the references to "1.16" below?